CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Features ✨ and improvements 🏁
 * Deprecated gesture settings `increaseRotateThresholdWhenPinchingToZoom` property. ([1632](https://github.com/mapbox/mapbox-maps-android/pull/1632))
+* Add experimental line-edge-clip property. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
+* Introduce `location-indicator-opacity` to location indicator layer that allows to control layer opacity. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
```

Comment:
Do we need this entry? Similar will be used when our platform code will land

Reply:
yeah, will remove for now